### PR TITLE
Mention original owner and plugin ID

### PR DESCRIPTION
**Pull request by Marcono1234**
See https://github.com/GradleUp/shadow/pull/928#issuecomment-2257050924
Existing users of the plugin might still remember the old owner and plugin ID, so this should hopefully reduce confusion and give further insight into the change.

Any feedback, especially regarding formatting or wording, is appreciated!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,13 @@
 
 Gradle plugin for creating fat/uber JARs with support for package relocation.
 
+> [!NOTE]\
```
Comment (Marcono1234) [inline README.md:5]: This is a GitHub "alert" section, see [documentation](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts).

File: README.md [hunk 2]
```diff
@@ -2,6 +2,13 @@
...
+> Previously this plugin was developed by [@johnrengelman](https://github.com/johnrengelman) and published under the ID `com.github.johnrengelman.shadow`
```
Comment (Goooler) [inline README.md:6]:
```suggestion
> Previously this plugin was developed by [@johnrengelman](https://github.com/johnrengelman) and published under the ID [`com.github.johnrengelman.shadow`](https://plugins.gradle.org/plugin/com.github.johnrengelman.shadow)
```

**Review by Goooler — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -2,6 +2,13 @@
...
+> before maintenance was transferred to the [GradleUp organization](https://github.com/GradleUp) to ensure future development, see [#908](https://github.com/GradleUp/shadow/issues/908).
+>
+> If you are still using the old plugin ID in your build script, we recommend to switch to the [new plugin ID `com.gradleup.shadow`](https://plugins.gradle.org/plugin/com.gradleup.shadow)
```
Comment (Goooler) [inline README.md:9]:
```suggestion
> If you are still using the old plugin ID in your build script, we recommend to switch to the new plugin ID [`com.gradleup.shadow`](https://plugins.gradle.org/plugin/com.gradleup.shadow)
```

**Comment by Marcono1234** [comment]
I have applied the changes you proposed also to the `src/docs/README.md` for consistency.

**Comment by Goooler** [comment]
Thanks!

**Pull request merged**